node-tests/blueprints/route-test.js

```diff
@@ -1058,27 +559,5 @@ describe('Blueprint: route', function () {
         );
       });
     });
-
-    describe('ember-page-title is installed', function () {
```

Comment:
Why are these `ember-page-title` related tests deleted (are they brought back in another file somewhere)?

Reply:
Since `ember-page-title` is included by default now, I deleted these tests and instead added page title-related assertions to *all* of the route tests since all routes generate templates with page title anyway.

Reply:
Shouldn't we still have tests that verify that the `page-title` helper is not present when `ember-page-title` is not installed? I could be wrong, but it seems all tests now assume the helper is always present?

Reply:
Hmm, I'm not sure. Since `ember-page-title` is installed by default, and these are blueprint tests, the blueprints in a new Ember app *will* always have page-title present unless people go and remove it after the fact. I guess we could still include a test or two for the case where it isn't installed just to make sure that it doesn't break if users remove it.

Reply:
I think having at least one test that still verifies the absence of the helper when `ember-page-title` is not installed is a good idea. These blueprints will also be used by people who upgrade their app and might have deleted `ember-page-title` in the past because e.g. they use `ember-cli-head` or a different approach. Just a suggestion though! Someone from the core team should decide.

Reply:
Ya, agree with @bertdeblock here. If the project doesn't have ember-page-title we need to not add helper invocations in the blueprint.